experimental/devices/mcp9808/example_test.go

```diff
@@ -8,12 +8,14 @@ import (
 	"fmt"
 	"log"
 
+	"periph.io/x/periph/conn/physic"
+
```

Comment:
Don't include an empty line here, did you add the import manually or did you use `goimports -w`?

Reply:
always use go imports, don't think I have ever used the -w switch